src/control/run_go_tests.sh

```diff
@@ -58,6 +58,13 @@ function setup_environment()
 	CGO_CFLAGS+="${SL_SPDK_PREFIX+ -I${SL_SPDK_PREFIX}/include}"
 	CGO_CFLAGS+="${SL_OFI_PREFIX+ -I${SL_OFI_PREFIX}/include}"
 	CGO_CFLAGS+="${SL_ISAL_PREFIX+ -I${SL_ISAL_PREFIX}/include}"
+	CGO_CFLAGS+="${SL_PMDK_PREFIX+ -I${SL_PMDK_PREFIX}/include}"
+
+	src_include="$(readlink -f $(pwd)/src/include)"
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]

Reply:
I checked out this PR to play around with it and realized that using $(pwd) will break most uses of this script, because it assumes that $(pwd) will evaluate to the top of the repo, which is often not a safe assumption. Earlier in this function, we find the top of the repo using the find_build_source function. So instead of using $(pwd), use this:

```suggestion
	src_include="$(dirname "$build_source")/src/include"
```

This correctly resolves to the top of the repo and also fixes the shellcheck warning.